docs_user/modules/proc_retrieving-topology-specific-service-configuration.adoc

```diff
@@ -52,11 +52,13 @@ endif::[]
     mysql -rsh "$SOURCE_MARIADB_IP" -uroot -p"$SOURCE_DB_ROOT_PASSWORD" -e 'SHOW databases;')
 echo "$PULL_OPENSTACK_CONFIGURATION_DATABASES"
 ----
+endif::[]
```

Comment:
I am not really sure about the placement, but it surely is missing

Reply:
this probably needs to go some lines above, as we need to have MARIADB_CLIENT_ANNOTATIONS defined for ospdo case